src/Server.cpp

```diff
@@ -235,6 +235,56 @@ bool cServer::InitServer(cSettingsRepositoryInterface & a_Settings, bool a_Shoul
 
 
 
+bool cServer::RegisterForgeMod(AString & a_ModName, AString & a_ModVersion, UInt32 a_ProtocolVersionNumber)
+{
+	auto & Mods = RegisteredForgeMods(a_ProtocolVersionNumber);
+
+	return Mods.insert({a_ModName, a_ModVersion}).second;
+}
+
+
+
+
+
+void cServer::UnregisterForgeMod(AString &a_ModName, UInt32 a_ProtocolVersionNumber)
```

Comment:
Please change to `const`, tolua returns params that AString references. Marking it as const avoid it.

Reply:
Wow, I've missed that one (and the style issue). Good thing I'm not the only reviewer :)

Reply:
Sorry I cheated, APIFuzzing...

Reply:
Anything that discovers problems is good, even better if it's automatic :)
Might be reasonable to add APIFuzzing to the CI builds.

Reply:
@madmaxoft Created a forum post

Reply:
Added `const` to the AString references in both RegisterForgeMod and UnregisterForgeMod